ocean_lib/models/v4/test/test_swap_fees.py

```diff
@@ -0,0 +1,3906 @@
+    another_consumer_wallet,
+):
+    """
+    Owner deploys a new ERC721 contract
+    """
+    erc721_factory = ERC721FactoryContract(
+        web3, get_address_of_type(config, "ERC721Factory")
+    )
+    tx = erc721_factory.deploy_erc721_contract(
+        "NFT",
+        "SYMBOL",
+        1,
+        ZERO_ADDRESS,
+        "https://oceanprotocol.com/nft/",
+        factory_deployer_wallet,
+    )
+    tx_receipt = web3.eth.waitForTransactionReceipt(tx)
+
+    event = erc721_factory.get_event_log(
+        erc721_factory.EVENT_NFT_CREATED,
+        tx_receipt.blockNumber,
+        web3.eth.block_number,
+        None,
+    )
+
+    assert event is not None
+
+    token_address = event[0].args.newTokenAddress
+    erc721_token = ERC721Token(web3, token_address)
+
+    assert erc721_token.balance_of(factory_deployer_wallet.address) == 1
+
+    erc721_token.add_manager(another_consumer_wallet.address, factory_deployer_wallet)
+    erc721_token.add_to_725_store_list(consumer_wallet.address, factory_deployer_wallet)
+    erc721_token.add_to_create_erc20_list(
+        consumer_wallet.address, factory_deployer_wallet
+    )
+    erc721_token.add_to_metadata_list(consumer_wallet.address, factory_deployer_wallet)
+
+    permissions = erc721_token.get_permissions(consumer_wallet.address)
+
+    assert permissions[1] == True
+    assert permissions[2] == True
+    assert permissions[3] == True
+
+
+def test_pool_ocean(
+    web3,
+    config,
+    factory_deployer_wallet,
+    consumer_wallet,
+    publisher_wallet,
+    factory_router,
+):
+    """Tests pool with ocean token and market fee 0.1%"""
+    erc721_factory = ERC721FactoryContract(
+        web3, get_address_of_type(config, "ERC721Factory")
+    )
+    side_staking = SideStaking(web3, get_address_of_type(config, "Staking"))
+    erc721_token = deploy_erc721_token(
+        config, web3, factory_deployer_wallet, consumer_wallet
+    )
+
+    # * Tests consumer deploys a new erc20DT, assigning himself as minter
+    cap = web3.toWei(100000, "ether")
+    tx = erc721_token.create_erc20(
+        ErcCreateData(
+            1,
+            ["ERC20DT1", "ERC20DT1Symbol"],
+            [
+                consumer_wallet.address,
+                factory_deployer_wallet.address,
+                consumer_wallet.address,
+                "0x0000000000000000000000000000000000000000",
+            ],
+            [cap, 0],
+            [],
+        ),
+        consumer_wallet,
+    )
+    tx_receipt = web3.eth.waitForTransactionReceipt(tx)
+    event = erc721_factory.get_event_log(
+        erc721_token.EVENT_TOKEN_CREATED,
+        tx_receipt.blockNumber,
+        web3.eth.block_number,
+        None,
+    )
+    erc20_address = event[0].args.newTokenAddress
+    erc20_token = ERC20Token(web3, erc20_address)
+
+    assert erc20_token.get_permissions(consumer_wallet.address)[0] == True
+
+    swap_fee = web3.toWei(0.001, "ether")
+    swap_ocean_fee = web3.toWei(0.001, "ether")
```

Comment:
Remove those swap fees variables, because you already use those values directly for `PoolData`.

Reply:
Those variables are being used for comparison.